docs(data table): audit fixes (#2042)
carbon-design-system/carbon-website: src/pages/components/data-table/usage.mdx

### Search

#### Collapsed search

A search field can be triggered through an icon button in the data table
toolbar. By default the search functionality follows the
[active search](/patterns/search-pattern/#active-search) pattern. The search
should be closed by default, and placed below the table title.

<Row>
<Column colLg={8}>

![data table search](images/datatable_search.gif)

</Column>
</Row>

#### Open search

> The search field can also appear always open on a data table. The field is

Reviewer: ```suggestion
The search field can appear always open on a data table. The field is
```

> The search field can appear always open on a data table. The field is
> placed on the left side of the table under the table title and stretches until
> it mets the actions on the right.

Reviewer 2: ```suggestion
The search field can always appear open on a data table. The field is
placed on the left side of the table under the table title and stretches until
it meets the actions on the right.
```

</Column>
</Row>

### Radio selection

> Instead of a multi-select checkbox function the data table also supports a

Reviewer: ```suggestion
Instead of a multiselect checkbox function, the data table supports a
```

single selection radio button control. Only one row can be selected at a time
when using the radio selection. The radio button is placed in the first column
on the left side.

<Row>
<Column colLg={12}>

![data table multi select](images/datatable-radio-select.png)

</Column>
</Row>

### Inline actions

Inline actions are functions that may be performed on a specific table row. Each
row is accompanied by
an [overflow menu](https://www.carbondesignsystem.com/components/overflow-menu/code) that
contains actions related specifically to that row.

> When the overflow menu contains less than three options, keep the actions inline

Reviewer: ```suggestion
When the overflow menu contains fewer than three options, keep the actions inline
```
